tests/testcases/core/domain/services/graphql/connections/PriceTypeConnectionQueriesTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace EventEspresso\tests\testcases\core\domain\services\graphql\connections;
+
+use EE_Dependency_Map;
+use EE_Error;
+use EEM_Price_Type;
+use EventEspresso\tests\testcases\core\domain\services\graphql\GraphQLUnitTestCase;
+
+class PriceTypeConnectionQueriesTest extends GraphQLUnitTestCase
+{
+    public $subscriber;
+
+    public function setUp()
+    {
+        parent::setUp();
+        if (PHP_VERSION_ID < 70000) {
+            $this->markTestSkipped(
+                'WP GraphQL compatible with PHP 7+ only'
+            );
+            return;
+        }
+
+        $this->model = EEM_Price_Type::instance();
+
+        $this->subscriber       = $this->factory()->user->create(
+            [
+                'role' => 'subscriber',
+            ]
+        );
+
+        $this->app_context = new \WPGraphQL\AppContext();
+
+        $this->app_info = new \GraphQL\Type\Definition\ResolveInfo(array());
+    }
+
+    public function tearDown()
+    {
+        parent::tearDown();
+    }
+
+    /**
+     * Creates several entities for use in cursor query tests
+     *
+     * @param  int $count Number of entities to create.
+     *
+     * @return array
+     */
+    public function create_entities($count = 20)
+    {
+
+        // Create entities
+        $created_entities = [];
+        for ($i = 1; $i <= $count; $i ++) {
+            $created_entities[ $i ] = $this->new_model_obj_with_dependencies('Price_Type');
+        }
+
+        return $created_entities;
+    }
```

Comment:
another good candidate for parent class logic, but with the model name passed as a parameter:

```suggestion
    public function create_entities($model_name, $count = 20)
    {
        // Create entities
        $created_entities = [];
        for ($i = 1; $i <= $count; $i ++) {
            $created_entities[ $i ] = $this->new_model_obj_with_dependencies($model_name);
        }
        return $created_entities;
    }
```